sdk/scheduler/src/main/java/com/mesosphere/sdk/offer/evaluate/placement/MarathonConstraintParser.java

```diff
@@ -44,58 +44,45 @@ private MarathonConstraintParser() {
     }
 
     /**
```

Comment:
I put it back in and updated its method signature, but I'm a little uncomfortable modifying code that currently is not touched by either the codebase or the test suite. I see how it can be used in that HDFS scenario (HDFS-412 ?) so I'll offer to work on that.